src/xopen/__init__.py

```diff
@@ -295,9 +317,6 @@ def _open_bz2(filename, mode):
 
 
 def _open_xz(filename, mode):
-    if lzma is None:
```

Comment:
Removing this is unrelated and should have been in a separate PR (but it’s fine now).

Reply:
I put it back in. It should be there. It should also be there when a potential `PipedXzWriter` is used but for some reason is not available.